src/Services/RefundsService.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Liuv\Larapix\Services;
+
+use GuzzleHttp\Client;
+use GuzzleHttp\Exception\ClientException;
+use Liuv\Larapix\Contracts\Features\RefundsContract;
+use Liuv\Larapix\ValueObjects\Refund;
+
+class RefundsService implements RefundsContract
+{
+
+    const BASE_API = 'https://api.openpix.com.br/api/openpix/v1';
+
+    private $client;
+
+    public function __construct(Client $client)
+    {
+        $this->client = $client;
+    }
+
+
+    public function findById(string $refundId): array
+    {
+        $uri = sprintf(self::BASE_API . '/refund/%s', $refundId);
+
+        $response = $this->client->get($uri);
+
+        return json_decode($response->getBody(), true);
+    }
+
+    public function findAll(array $parameters = []): array
+    {
+        $uri = self::BASE_API . '/refund';
+
+        $response = $this->client->get($uri);
+
+        return json_decode($response->getBody(), true);
+    }
+
+    public function create(Refund $refund): array
+    {
+        
+        $uri = self::BASE_API . '/refund';
+        $response = $this->client->post($uri, [
+            'headers' => [
+                'Content-Type' => 'application/json',
+                'Accept' => 'application/json',
+            ],
+            'body' => json_encode($refund)
+        ]);
+        // TODO: decide how to differentiate any type of error
```

Comment:
About how to differentiate errors. We can create an Adapter using error message parsing with some kind of Factory class, so that when they update how the error is displayed we can change it easily.

Reply:
I'm talking with @seraphini and @daniloab about this issue. On the payload response I don't have any kind of ID to get things easier to parse. 

Now I'll have to figure out how many errors it can retrieve and parse one by one and it's not a good approach... Probably will wait until get some answer if it will be a priority there since we have time until release the package.

Reply:
Got it. It makes sense, since the package is not released yet.

Reply:
we gonna improve errors in the next API design work

Reply:
@sibelius nice to know! For now we will just repass the API error through the client. 

Let me know when you guys finish it and also if need any help on design ping me.